slices/systemd.yaml

```diff
@@ -0,0 +1,691 @@
+      /usr/lib/systemd/system/sysinit.target.wants/dev-hugepages.mount:
+      /usr/lib/systemd/system/sysinit.target.wants/dev-mqueue.mount:
+      /usr/lib/systemd/system/sysinit.target.wants/integritysetup.target:
+      /usr/lib/systemd/system/sysinit.target.wants/kmod-static-nodes.service:
+      /usr/lib/systemd/system/sysinit.target.wants/ldconfig.service:
+      /usr/lib/systemd/system/sysinit.target.wants/proc-sys-fs-binfmt_misc.automount:
+      /usr/lib/systemd/system/sysinit.target.wants/sys-fs-fuse-connections.mount:
+      /usr/lib/systemd/system/sysinit.target.wants/sys-kernel-config.mount:
+      /usr/lib/systemd/system/sysinit.target.wants/sys-kernel-debug.mount:
+      /usr/lib/systemd/system/sysinit.target.wants/sys-kernel-tracing.mount:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-ask-password-console.path:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-binfmt.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-firstboot.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-journal-catalog-update.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-journal-flush.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-journald.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-machine-id-commit.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-modules-load.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-pcrmachine.service:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-pcrphase-sysinit.service:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-pcrphase.service:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-random-seed.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-sysctl.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-sysusers.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup-dev-early.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup-dev.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-tpm2-setup-early.service:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-tpm2-setup.service:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-update-done.service:
+      /usr/lib/systemd/system/sysinit.target.wants/systemd-update-utmp.service:
+      /usr/lib/systemd/system/sysinit.target.wants/veritysetup.target:
+      /usr/lib/systemd/system/syslog.socket:
+      /usr/lib/systemd/system/system-systemd\x2dcryptsetup.slice:
+      /usr/lib/systemd/system/system-systemd\x2dveritysetup.slice:
```

Comment:
The backslashes are interesting. Are the files extracted properly?

Reply:
Yea - those are a bit odd, but they get extracted it seems